### ISSUE TITLE
[request] remove all midi connections from plugin

### ISSUE THREAD

**Issue by VirToReal** [LMMS/lmms#4385]
Yeah, thats a request, not a real issue:
I'm using a novation Launch Control XL and connect it's knobs to a plugin, for example Triple OSC. After generating a nice sound, I have to disconnect every midi knob in order to save my creation as a preset. Because the midi connection will be saved in the preset-file (which is really great), all knob positions will be set to zero. 

If there would be a button to erase all connections, I can save one preset as midi-preset, using this for creating sounds via launch control, deleting all midi connections and save it as a real preset.

Maybe two file-formats? One for midi-connected presets, and one for non-midi-connected presets?

**Comment by PhysSong** [comment]
Are there more use-cases for "remove all midi connections"? If preset saving is the only one, I can't find a reason for that button.
> Because the midi connection will be saved in the preset-file

Related: #2625

**Comment by VirToReal** [comment]
Not really... just leaving the last midi-value on the knob while removing the connection would be great. Removing every connection would be nice, if there are, for example on the triple oscillator, all knobs connected to a midi-controller. Or maybe just a keyboard shortcut for removing the midi-connection while pressing the connected button/knob.

In each case of course, the midi-value should be placed on the actual value of the knob/button while removing the midi-connection.
